src/translations/sqlb_ar_SA.ts

```diff
@@ -3365,9 +3454,9 @@ You can drag SQL statements from the Schema column and drop them into the SQL ed
         <translation>احفظ ملف SQL</translation>
     </message>
     <message>
-        <location filename="../MainWindow.cpp" line="282"/>
+        <location filename="../MainWindow.cpp" line="291"/>
         <source>Ctrl+E</source>
-        <translation></translation>
+        <translation>Ctrl+E</translation>
```

Comment:
Shortcuts should not be translated.
See the [wiki](https://github.com/sqlitebrowser/sqlitebrowser/wiki/Translations#dont-translate-key-names-in-shortcuts) for further information.